libcontainer/specconv/spec_linux.go

```diff
@@ -462,11 +460,16 @@ func initSystemdProps(spec *specs.Spec) ([]systemdDbus.Property, error) {
 		if err != nil {
 			return nil, fmt.Errorf("Annotation %s=%s value parse error: %w", k, v, err)
 		}
-		if isSecSuffix(name) {
-			name = strings.TrimSuffix(name, "Sec") + "USec"
-			value, err = convertSecToUSec(value)
-			if err != nil {
-				return nil, fmt.Errorf("Annotation %s=%s value parse error: %w", k, v, err)
+		// Check for Sec suffix.
+		if trimName := strings.TrimSuffix(name, "Sec"); len(trimName) < len(name) {
+			// Check for a lowercase ascii a-z just before Sec.
+			if ch := trimName[len(trimName)-1]; ch >= 'a' && ch <= 'z' {
+				// Convert from Sec to USec.
+				name = trimName + "USec"
+				value, err = convertSecToUSec(value)
+				if err != nil {
+					return nil, fmt.Errorf("Annotation %s=%s value parse error: %w", k, v, err)
```

Comment:
Not something new, so probably better to fix separately, but this should've been lowercase

Reply:
Are you aware of any linter for such things? Would be nice to have

Reply:
I expect the `revive` linter to be warning about it; https://github.com/opencontainers/runc/pull/2998/commits/2cca1149cd9af24f3535d38365c0e48c93bbc6e6

Reply:
(that's a commit from https://github.com/opencontainers/runc/pull/2998)